src/view/EverlinkPages.tsx

```diff
@@ -28,48 +66,67 @@ export default function EverlinkPages() {
                 sm: '70%',
                 xs: '85%',
               },
-              display: 'flex',
-              flexDirection: 'column',
-              mx: 'auto',
-              mt: '74px',
-              bgcolor: 'primary.contrastText',
-              p: { md: '40px 24px', xs: '24px 16px' },
-              borderRadius: 3,
-              boxShadow: '0px 4.03px 12.89px 0px #080F340F',
+              ...styles.mainBox,
             }}
           >
             {' '}
             <Box sx={{ borderBottom: '1px solid #D9DBE9', pb: 2, mb: 3 }}>
               <LinearStepper activeStep={activeStep} />
             </Box>
-            {activeStep === 0 ? (
-              <>
-                <Form
-                  userName={userName}
-                  setUserName={setUserName}
-                  bio={bio}
-                  setBio={setBio}
-                  setActiveStep={setActiveStep}
-                  analyticsTag={analyticsTag}
-                  setAnalyticsTag={setAnalyticsTag}
-                  urlButtons={urlButtons}
-                  setUrlButtons={setUrlButtons}
-                />
-              </>
-            ) : (
-              <h1>Select Theme</h1>
-            )}
+            {(() => {
```

Comment:
Move it outside of the return statement just like you did for the `renderThemePreview`

Reply:
Resolved